Update ScientificCalculator

Sjan00 opened the pull request:
Added frame and buttons

Sjan00 committed:
Update ScientificCalculator

Added frame and buttons

MaiNgo95 commented:
The method addActionListener(ActionListener) in the type AbstractButton is not applicable to the arguments (ScientificCalculator).

MaiNgo95 reviewed (changes requested):
Please add the java file. Need to improve by adding "implements ActionListener."

Sjan00 committed:
Update ScientificCalculator

Sjan00 commented:
I have updated the file, I added "implements ActionListener"
I've also updated line 22, from 
     frame = new JFrame("Calculator") to frame = new JFrame("Scientific Calculator")

Sjan00 committed:
Update ScientificCalculator

Updated the buttons.  Added new function and function buttons.
